src/index.css

```diff
@@ -2,3 +2,10 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400&display=swap');
+
+body {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 16px;
+}
```

Comment:
- [ ] As it is a requirement for this project, and a required skill as a front end dev to stick to the  design, please kindly update your app to match with the design. Here are things to pay attention to; 
- The middle column in design is missing in your implementation(see screenshot below)
- The book `title input field`, `category field` and `add button` are not aligned(see screenshot below).

***The design***
![theD](https://user-images.githubusercontent.com/65852621/150312522-a967d2a0-a4e4-4d84-ae8d-cd8e8b374d76.png)

***Your implementation***
![yourI](https://user-images.githubusercontent.com/65852621/150312546-890ab740-8d44-4dd5-bcf1-fd98817eb77c.png)

Reply:
Sure bro.
I have fixed things up as required and now the design looks just as required.